src/platform/plugins/shared/data_view_field_editor/public/components/field_editor/field_editor.tsx

```diff
@@ -296,6 +297,7 @@ const FieldEditorComponent = ({ field, onChange, onFormModifiedChange, isDisable
       {field?.parentName && (
         <>
           <EuiCallOut
+            announceOnMount
```

Comment:
```suggestion
            announceOnMount={false}
```
I don't think this one toggles with user interaction.